src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -258,6 +258,7 @@ public void BuildBasicApplicationReleaseProfiledAotWithoutDefaultProfile ()
 				AotAssemblies = true,
 				PackageName = "com.xamarin.buildaotappandbundlewithspecialchars",
 			};
+			proj.SetProperty ("AndroidNdkDirectory", AndroidNdkPath);
```

Comment:
Why is this property added, when it (apparently?) wasn't previously required?  Ditto elsewhere.

Reply:
`$(AndroidNdkDirectory)` is not passed into the MSBuild tests anymore, I deleted:

https://github.com/xamarin/xamarin-android/blob/5dcf2945e4cd7da50c5c8f76608e0d0266022b09/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs#L314-L317

And so we need to pass it in, for tests using LLVM or `$(BundleAssemblies)` now. We should also have some confidence that the other tests work now *without* `$(AndroidNdkDirectory)` passed in.